src/coffee/mappings.coffee

```diff
@@ -88,11 +97,6 @@ class CustomFieldMappings
       @errors.push "[row #{rowIndex}:#{typeDefinitionKey}] Can not parse money '#{rawMoney}'!"
       return
 
-    validCurr = CONS.REGEX_CUR.exec matchedMoney[1]
```

Comment:
The API already checks if the currency is valid, I think having the API as the single source of truth is better. For one the definitions could get out of sync and it adds complexity that is not really needed. I do see the benefit of checking it instantly though so if wanted, I'm okay with adding it back.